proto/cometbft/abci/v2/types.proto

```diff
@@ -0,0 +1,278 @@
+}
+
+message RequestPrepareProposal {
+  // the modified transactions cannot exceed this size.
+  int64 max_tx_bytes = 1;
+  // txs is an array of transactions that will be included in a block,
+  // sent to the app for possible modifications.
+  repeated bytes            txs                  = 2;
+  ExtendedCommitInfo        local_last_commit    = 3 [(gogoproto.nullable) = false];
+  repeated Misbehavior      misbehavior          = 4 [(gogoproto.nullable) = false];
+  int64                     height               = 5;
+  google.protobuf.Timestamp time                 = 6 [(gogoproto.nullable) = false, (gogoproto.stdtime) = true];
+  bytes                     next_validators_hash = 7;
+  // address of the public key of the validator proposing the block.
+  bytes proposer_address = 8;
+}
+
+message RequestProcessProposal {
+  repeated bytes       txs                  = 1;
+  CommitInfo           proposed_last_commit = 2 [(gogoproto.nullable) = false];
+  repeated Misbehavior misbehavior          = 3 [(gogoproto.nullable) = false];
+  // hash is the merkle root hash of the fields of the proposed block.
+  bytes                     hash                 = 4;
+  int64                     height               = 5;
+  google.protobuf.Timestamp time                 = 6 [(gogoproto.nullable) = false, (gogoproto.stdtime) = true];
+  bytes                     next_validators_hash = 7;
+  // address of the public key of the original proposer of the block.
+  bytes proposer_address = 8;
+}
+
+//----------------------------------------
+// Response types
+
+message Response {
+  oneof value {
+    v1.ResponseException          exception            = 1;
+    v1.ResponseEcho               echo                 = 2;
+    v1.ResponseFlush              flush                = 3;
+    v1.ResponseInfo               info                 = 4;
+    ResponseInitChain             init_chain           = 6;
+    v1.ResponseQuery              query                = 7;
+    ResponseBeginBlock            begin_block          = 8;
+    ResponseCheckTx               check_tx             = 9;
+    ResponseDeliverTx             deliver_tx           = 10;
+    ResponseEndBlock              end_block            = 11;
+    v1.ResponseCommit             commit               = 12;
+    v1.ResponseListSnapshots      list_snapshots       = 13;
+    v1.ResponseOfferSnapshot      offer_snapshot       = 14;
+    v1.ResponseLoadSnapshotChunk  load_snapshot_chunk  = 15;
+    v1.ResponseApplySnapshotChunk apply_snapshot_chunk = 16;
+    ResponsePrepareProposal       prepare_proposal     = 17;
+    ResponseProcessProposal       process_proposal     = 18;
+  }
+  reserved 5;
+}
+
+message ResponseInitChain {
+  cometbft.types.v2.ConsensusParams consensus_params = 1;
+  repeated v1.ValidatorUpdate       validators       = 2 [(gogoproto.nullable) = false];
+  bytes                             app_hash         = 3;
+}
+
+message ResponseBeginBlock {
```

Comment:
I think because the `Event` type changed. Changes in types upon which a type is dependent has a "viral" effect in triggering the need for updates, so if the `Event` type changes in some breaking way, all types that depend on it need new versions.

If the `Event` type changes in a non-breaking way, or doesn't change, then this viral effect isn't present.

Reply:
Yes, it's the new `Event`.